src/client/pages/Checkout/CheckoutDetails/CheckoutDetails.tsx

```diff
@@ -2,22 +2,43 @@ import React from 'react'
 import { useTextKeys } from 'utils/textKeys'
 import { useQuoteCartIdFromUrl } from 'utils/hooks/useQuoteCartIdFromUrl'
 import { useCurrentLocale } from 'l10n/useCurrentLocale'
+import { useQuoteCartData } from 'utils/hooks/useQuoteCartData'
 import { CheckoutPageWrapper } from '../shared/CheckoutPageWrapper'
 import { Footer } from '../shared/Footer'
+import { StartDate } from '../../Offer/Introduction/Sidebar/StartDate'
+import { PaymentInfo } from '../shared/PaymentInfo'
+import { YourPlan } from './components/YourPlan/YourPlan'
+import { QuoteDetails } from './components/QuoteDetails/QuoteDetails'
+import { PageSection } from './components/PageSection'
+import { SubSection } from './components/SubSection'
 
 export const CheckoutDetails = () => {
   const textKeys = useTextKeys()
   const { path: localePath } = useCurrentLocale()
   const { quoteCartId } = useQuoteCartIdFromUrl()
-
+  const data = useQuoteCartData()
+  if (!data) {
+    return null
+  }
+  const priceData = data.priceData
+  const quoteDetails = data.quoteDetails
   const paymentPageLink = `/${localePath}/new-member/checkout/payment/${quoteCartId}`
-
   return (
     <CheckoutPageWrapper>
+      <PageSection>
+        <YourPlan {...priceData} />
+        <SubSection headlineText={textKeys.SIDEBAR_STARTDATE_CELL_LABEL()}>
+          <StartDate quoteCartId={quoteCartId} modal size="sm" />
```

Comment:
This start date components now takes up full width on desktop which looks a bit crazy 😅 Maybe would be good to set a max-width?

Reply:
ah. good catch. 👍